python_modules/dagster/dagster/_generate/templates/PROJECT_NAME_PLACEHOLDER/requirements.txt.tmpl

```diff
@@ -0,0 +1,3 @@
+-e .
```

Comment:
Do we need `requirements.txt` and `setup.py` for the default project? Not super clear on why someone would need both

Reply:
requirements.txt will be used for deployment which is more tied to a concrete environment. here's a convo in slack [here](https://elementl-workspace.slack.com/archives/C03A0D72A6T/p1658515403719279)

Reply:
maybe some in file comments about why both are included?